.github/dependabot.yml

```
@@ -4,6 +4,15 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    # jqwik is pinned to the newest 1.9.x in pom.xml and renovate.json: the 1.10.x
+    # engine is not built for the JUnit 6 platform Ares declares directly and fails
+    # deterministically with an ArrayIndexOutOfBoundsException in
+    # StoreRepository.streamAllStores when Ares runs thread-spawning properties
+    # (JqwickTest). Mirror Renovate's pin here so Dependabot stops offering the bump.
+    # Revisit once jqwik ships a JUnit 6 compatible release.
+    ignore:
+      - dependency-name: "net.jqwik:jqwik"
+        versions: [ ">=1.10.0" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)